superset-frontend/src/SqlLab/components/LimitControl.tsx

```diff
@@ -30,15 +29,27 @@ import { t } from '@superset-ui/translation';
 
 import ControlHeader from '../../explore/components/ControlHeader';
 
-const propTypes = {
-  value: PropTypes.number,
-  defaultQueryLimit: PropTypes.number.isRequired,
-  maxRow: PropTypes.number.isRequired,
-  onChange: PropTypes.func.isRequired,
+export interface LimitControlProps {
+  value?: number;
+  defaultQueryLimit: number;
+  maxRow: number;
+  onChange: (value: number) => void;
+}
+
+interface LimitControlState {
+  textValue: string;
+  showOverlay: boolean;
+}
+
+const popoverStyle = {
```

Comment:
instead of this, let's make a `PopoverDiv` component with `styled` and apply the styling that way?